include/pybind11/detail/class.h

```diff
@@ -476,9 +479,8 @@ extern "C" inline int pybind11_clear(PyObject *self) {
 inline void enable_dynamic_attributes(PyHeapTypeObject *heap_type) {
     auto type = &heap_type->ht_type;
 #if defined(PYPY_VERSION) && (PYPY_VERSION_NUM < 0x06000000)
-    pybind11_fail(std::string(type->tp_name) + ": dynamic attributes are "
-                                               "currently not supported in "
-                                               "conjunction with PyPy!");
+    pybind11_fail(get_tp_type(type) + ": dynamic attributes are currently not "
```

Comment:
This still shows `get_tp_type`. Oversight?

Reply:
Oh. That was even wrong :-| It used to be `get_tp_name`... This is part of the code is going away in #2456, though. That's why it's not tested.